src/main/java/de/tum/in/www1/artemis/domain/notification/SingleUserNotificationFactory.java

```diff
@@ -44,4 +48,40 @@ public static SingleUserNotification createNotification(Post post, NotificationT
         }
         return notification;
     }
+
+    /**
+     * Creates an instance of PlagiarismNotification
+     *
+     * @param plagiarismComparisonId ID of the plagiarism comparison associated with this case.
+     * @param recipient              affected student
+     * @param author                 instructor creating notification
+     * @param message                instructors message
+     * @return the PlagiarismNotification
+     */
+    private static SingleUserNotification createPlagiarismNotification(Long plagiarismComparisonId, Long courseId, User recipient, User author, String message, boolean isUpdate) {
+        String title = isUpdate ? "Update on your plagiarism case" : "Possible case of Plagiarism";
+        if (recipient.getLangKey() != null) {
```

Comment:
We currently have no support for the translations (i.e. the German language) for the notifications. Everything is (and was for ~2+ years in English). I think it is a great idea but this should be done in a separate refactoring that adds translations to all notifications and not just to one + there is an issue with this translation method. The notification will be shown at the client side -> the translation should happen on the client side. + so currently if the User has set his language Key to German at the time the notification is created the notification will be in German but if the user changes his language dynamically in the client to English the notification will still be in German. So yeah. I would recommend to remove the translation.